File: Landowner/src/landowner/ui/console/ExitOperation.java
package landowner.ui.console;

public class ExitOperation implements OperationRole {

	private String message;

	public ExitOperation(String message) {
		this.message = message;
	}

	public void run() {
		System.out.println("Exited succesfully.");
	}

	public String getMessage() {
		return message;
	}

	public boolean doContinue() {
		return false;
	}

}

File: caphyon/source/ExitOperation.cpp
#include"../includes/ExitOperation.h"
#include<iostream>
using namespace std;

ExitOperation::ExitOperation(string message) {
	this->message = message;
}

void ExitOperation::run() {
	cout << "\nExited succesfully.\n";
}

string ExitOperation::getMessage() {
	return message;
}

bool ExitOperation::doContinue() {
	return false;
}